Source/santactl/Commands/sync/SNTCommandSyncRuleDownload.m

```diff
@@ -115,6 +115,8 @@ - (SNTRule *)ruleFromDictionary:(NSDictionary *)dict {
     newRule.type = SNTRuleTypeBinary;
   } else if ([ruleTypeString isEqual:kRuleTypeCertificate]) {
     newRule.type = SNTRuleTypeCertificate;
+  } else if ([ruleTypeString isEqual:kRuleTypePackage]) {
+    newRule.type = SNTRuleTypePackage;
```

Comment:
Ah crap, never mind. That would [choke](https://github.com/google/santa/blob/master/Source/santad/DataLayer/SNTRuleTable.m#L168) and stop processing the rules. We should filter package rules before L56 of SNTCommandSyncRuleDownload.m, then add back before L79.